drgn/cli.py

```diff
@@ -386,6 +386,12 @@ def _main() -> None:
         const="none",
         help="don't print any logs or download progress",
     )
+    parser.add_argument(
+        "-e",
+        dest="exec",
```

Comment:
Do we want a long option, too? I guess `python -c` doesn't have one, so I'm undecided.
```suggestion
        "-e",
        "--exec",
```

Reply:
I had thought about that for a bit before realizing that I didn't know exactly what `-e` even stands for...

- `--expression`, `--expr`
- `--evaluate`, `--eval`
- `--execute`, `--exec`

Three different words to choose from, each with a commonly used four-letter shortening. Technically `exec` is correct (the others imply only expressions, rather than statements, can be executed). But I'm not sure that distinction is obvious enough that `--execute` or `-e` stands out as correct. Especially given that you normally "execute" a script just like you'd "execute" a statement, so the most correct option doesn't even semantically highlight that the argument should be literal code, not a script.

So in my opinion, leaving it without a long option is best.